storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -493,7 +493,7 @@ bool get_mysql_vars(MYSQL *connection) {
   }
 
   if (slave_parallel_workers_var != NULL &&
-      atoi(slave_parallel_workers_var) > 0) {
+      atoi(slave_parallel_workers_var) > 1) {
```

Comment:
If someone wants to take a server backup with a version lower than 8.0.27, does 1 mean multiple threads or a single thread?